default/scripting/buildings/colonies/SP_BANFORO.focs.txt

```diff
@@ -45,18 +45,21 @@ BuildingType
         Population high = 0
         Not Planet environment = Uninhabitable species = "SP_BANFORO"
         Not Contains Building name = "BLD_COL_BANFORO"
-        ResourceSupplyConnected empire = Source.Owner condition = And [
-            Planet
-            OwnedBy empire = Source.Owner
-            Or [
-                And [
-                    Species name = "SP_BANFORO"
-                    Population low = [[MIN_RECOLONIZING_SIZE]]
-                    Happiness low = 5
-                ]
-                And [
-                    HasSpecial name = "EXTINCT_BANFORO_SPECIAL"
-                    Contains Building name = "BLD_XENORESURRECTION_LAB"
+        Or [
+            OwnerHasTech name = "TECH_COL_BANFORO"
+            ResourceSupplyConnected empire = Source.Owner condition = And [
+                Planet
+                OwnedBy empire = Source.Owner
+                Or [
+                    And [
+                        Species name = "SP_BANFORO"
+                        Population low = [[MIN_RECOLONIZING_SIZE]]
+                        Happiness low = 5
+                    ]
+                    And [
+                        HasSpecial name = "EXTINCT_BANFORO_SPECIAL"
+                        Contains Building name = "BLD_XENORESURRECTION_LAB"
+                    ]
```

Comment:
Hm, IIRC it has been @Deepsloth who designed and implemented the extinct species, but I *think* the idea is that you need some kind of "remnants" of members of the extinct species (represented by the special) and the lab to clone them, this way providing an alternative source of colonists (as you don't have an existing colony).

The knowledgle alone isn't sufficient. You need sufficient "raw material" to work with. Hence the necessity to have supply connection to a colony with the "raw material" and the facility to make colonists out that.

Personally I prefer that approach, this way you can loose the species again if you loose all their colonies as well as the planet where you found their remnants originally. More interesting IMO. 

The other approach would mean once you got the tech, you can just recreate them whenever and wherever you want. Which is something you can't even do with normal species, so this strikes me as quite unbalanced.